src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -543,7 +575,13 @@ subroutine mixedlayer_restrat_OM4(h, uhtr, vhtr, tv, forces, dt, h_MLD, VarMix,
   !$OMP do
   do J=js-1,je ; do i=is,ie
     u_star = max(CS%ustar_min, 0.5*(U_star_2d(i,j) + U_star_2d(i,j+1)))
-
+    ! Compute I_LFront = 1 / (frontal length scale) [m-1]
+    lfront = 0.5 * (mle_fl_2d(i,j) + mle_fl_2d(i,j+1))
```

Comment:
I think `mle_fl_2d(i,j)` is either out-of-bounds or uninitialized when j=js-1. This is probably contributing to the dimensional scaling failure